library/core/src/main/java/com/google/android/exoplayer2/text/ssa/SsaDecoder.java

```diff
@@ -319,6 +321,27 @@ private static Cue createCue(
             style.fontSize / screenHeight,
             Cue.TEXT_SIZE_TYPE_FRACTIONAL_IGNORE_PADDING);
       }
+      if (style.bold && style.italic) {
+        spannableText.setSpan(
+            new StyleSpan(Typeface.BOLD_ITALIC),
+            0,
+            spannableText.length(),
+            SpannableString.SPAN_EXCLUSIVE_EXCLUSIVE);
+      }
+      else if(style.bold){
```

Comment:
I wouldn't put the `else` in new lines, to be consistent with the rest of the code. Also if we want to make it really consistent I'd add these trailing comments `/* start= */` and `/* end= */` at the start of the 2nd and 3rd params of `setSpan` - similarly how it is done for the primary color span.

And lastly, you have a couple of missing whitespaces, e.g. `if(style.bold){` -> `if (style.bold) {`

Reply:
Thank you. I will fix this.

Reply:
I agree with @szaboa - but don't worry too much about the styling/whitespace. When we merge PRs we run them through https://github.com/google/google-java-format (same for all the code we author ourselves), which sorts all this stuff out without needing to think too hard about it.